src/views/InteractionDesigner.vue

```diff
@@ -193,23 +195,31 @@ export default {
 
   /** @note - mixin's mount() is called _before_ local mount() (eg. InteractionDesigner.legacy::mount() is 1st) */
   mounted() {
+      this.flow_setActiveFlowId({flowId: this.id})
+
+      //if nothing was found for the flow Id
+      if(!this.activeFlow) {
+        this.flow_setActiveFlowId({flowId: null})
+        this.$router.replace(`${this.route('flows.fetchFlow', {flowId: this.id})}?nextUrl=${this.$route.path}`)
```

Comment:
I believe we can get the router to `querystring` serialize our query params for us. Something like this should ensure safely urlencoded param value:
```
const {id: flowId} = this
const {path: nextUrl} = this.$route

this.$router.replace(
    this.route('flows.fetchFlow', {flowId}), 
    {query: {nextUrl}})
```